sections/dashboard/EligibilityBox/EligibilityBox.tsx

```diff
@@ -0,0 +1,82 @@
+import { useCallback, useState } from 'react';
+import styled from 'styled-components';
+
+import Button from 'components/Button';
+import Input from 'components/Input/Input';
+import Text from 'components/Text';
+
+import EligibilityMap from './EligibilityMap';
+
+const EligibilityBox = () => {
+	const [address, setAddress] = useState('');
+	const [isEligible, setIsEligible] = useState<boolean>();
+
+	const handleChangeAddress = useCallback((e) => {
+		setAddress(e.target.value);
+	}, []);
+
+	const handleCheck = useCallback(() => {
+		setIsEligible(!!(!!address && EligibilityMap[address]));
+	}, [address]);
+
+	return (
+		<div>
+			<EligibilitySectionTitle>SNX Staker Distribution</EligibilitySectionTitle>
+			<EligibilityContainer>
+				<EligibilityText>Enter your wallet address to check eligibility</EligibilityText>
+				<EligibilityInput placeholder="0x..." value={address} onChange={handleChangeAddress} />{' '}
+				<Button fullWidth size="sm" onClick={handleCheck}>
+					Check
```

Comment:
Same here, should translate unless it's temporary

Reply:
Same as above